test/browser/specs/update-snapshot.test.mjs

```diff
@@ -0,0 +1,49 @@
+import assert from 'node:assert'
+import fs from 'node:fs'
+import test from 'node:test'
+import { startVitest } from 'vitest/node'
+
+let vitest
+
+test.after(async () => {
+  // force exitCode mutated by vitest
+  process.exitCode = 0
```

Comment:
This seems sketchy. What if `exitCode` was mutated by `node:test`?

Reply:
I didn't investigate much but I meant to do something similar as `runVitest` test util here:

https://github.com/vitest-dev/vitest/blob/044c7386381c9f74aaf493c42e4f9af8c22a1603/test/test-utils/index.ts#L40-L42

I'll look into more detail what exactly is happening with exit code.

Reply:
I refined some assertions to be more precise about `process.exitCode` https://github.com/vitest-dev/vitest/pull/4867/commits/d2366e8cbe9610d6d5c1928d482d3587d60518f7

> Why Vitest mutates exitCode?

Vitest sets `process.exitCode = 1` whenever test failed during `runtFiles`:

https://github.com/vitest-dev/vitest/blob/d2366e8cbe9610d6d5c1928d482d3587d60518f7/packages/vitest/src/node/core.ts#L505-L506